controllers/unavailability_update.py

```diff
@@ -0,0 +1,55 @@
+from flask import Blueprint, request, jsonify
```

Comment:
This file should be in v2/unavailability, and it should be called api. Please merge in main, integrate your file into the existing api.py and replicate the changes I made in my model PR #161. Everything should be the same except for the line ``v2_api.add_resource(VolunteerUnavailabilityV2, '/v2/volunteers/', '/v2/volunteers/<user_id>/unavailability/<event_id>')``. You'll want yours to be registered with ``v2_api.add_resource(CreateVolunteerUnavailabilityV2, '/v2/volunteers/', '/v2/volunteers/<user_id>/unavailability')``.

Reply:
It's fine if you're not quite sure what your doing or why. Ask me for help if needed and I'll cover it in more detail whenever I meet with the backend team :)